### ISSUE TITLE
i686: Undefined reference to __restore_rnd

### ISSUE THREAD

**Issue by tuliom** [libdfp/libdfp#115]
Reproduced on commit fd8eb0b16d823 with:
```
configure --host=i686-linux-gnu --with-backend=libdecnumber CFLAGS="-m32" CXXFLAGS="-m32"
```

Error message:
```
expd128.os: In function `__expd128':
expd128.c:(.text+0xc77): undefined reference to `__restore_rnd'
collect2: error: ld returned 1 exit status
make: *** [Makefile:226: libdfp-1.0.14.so] Error 1
```

Does sysdeps/generic/dfp_inline.h need to use `__attribute__((always_inline))`?

**Comment by pmur** [comment]
Yes, those annotations should be present. #114 addresses this, and a clean i386 build runs and tests happily now.  `$SRCDIR/configure --host=i386-linux CC="gcc -m32" CXX="g++ -m32"`

**Comment by pmur** [comment]
This should be resolved now that #114 is merged.
